src/style.css

```
@@ -0,0 +1,76 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: "Avenir-Black";
```

font-family should be same, but with different font-weight
```suggestion
  font-family: "Avenir";
```